src/components/Button.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import '../styles/Button.css';
+
+class Button extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+
+    };
+  }
+
+  render() {
+    const { value } = this.props;
+    const { classStyle } = this.props;
```

Comment:
I think we could have this in one line `const { value, classStyle } = this.props;`